### PR TITLE
Add possibility to define disabled days

### PR DESCRIPTION
**Pull request by kvolkovich-sc**
API changes
===

Added new property `modifiers`.
Implemented default `disabled` modifier.
Disabled days validation and display for `DateInput` and `DateRangeInput` components can be implemented by validation at parent component level and passing validation result to `isValid` property.
For validation use `import { ModifiersUtils } from @opuscapita/react-dates`.
See component documentation for details.

### CODE REVIEW

File: src/client/components/DatePicker/DatePicker.react.js [hunk 1]
```diff
@@ -68,9 +70,11 @@ class DatePicker extends Component {
     this.hidePicker();
   };
 
-  handleDayClick = day => {
-    this.handleDateChange(day);
-    this.setState({ showPicker: false });
+  handleDayClick = (value, modifiers) => {
+    if (!modifiers.disabled) {
+      this.handleDateChange(value);
+      this.setState({ showPicker: false });
```
Comment (estambakio-sc) [inline src/client/components/DatePicker/DatePicker.react.js]: Better be `this.hidePicker();` instead.

File: src/client/components/DateRangeInput/DateRangeInput.DOCUMENTATION.md [hunk 2]
```diff
@@ -41,6 +42,16 @@ Allows select date range using mouse.
...
 />
+
+<hr />
+
+<DateRangeInput
+  dateFormat="dd.MM.yyyy"
+  locale={'de'}
+  onChange={_scope.handleChange1.bind(_scope)}
+  value={_scope.state.value1}
+  disabledDays={{ after: new Date() }}
```
Comment (estambakio-sc) [inline src/client/components/DateRangeInput/DateRangeInput.DOCUMENTATION.md]: Props reference and example don't match: `modifiers` vs. `disabledDays`.

File: src/client/components/DayPicker/DayPicker.DOCUMENTATION.md [hunk 3]
```diff
@@ -32,7 +32,17 @@ See react-day-picker [methods reference](http://react-day-picker.js.org/APIMetho
 <DayPicker
   dayPickerRef={ref => (window.picker = ref)}
   onChange={ day => console.log('day:', day) }
-  locale="de"
+  locale="en"
+  disabledDays={{ after: new Date() }}
```
Comment (estambakio-sc) [inline src/client/components/DayPicker/DayPicker.DOCUMENTATION.md]: Wrong naming? No such prop in prop types neither in component's code.

File: src/client/components/DayPicker/DayPicker.react.js [hunk 4]
```diff
@@ -5,33 +5,38 @@ import "react-day-picker/lib/style.css";
 import localeUtils from '../../dayjs/reactDayPickerUtils';
 import getMessage from '../translations';
 import { splitProps, zeroTime } from '../utils';
+import dayjs from 'dayjs';
 import './DayPicker.less';
+window.dayjs = dayjs;
```
Comment (estambakio-sc) [inline src/client/components/DayPicker/DayPicker.react.js]: Obsolete.

File: src/client/components/DatePicker/DatePicker.react.js (hunk 1 above)
Comment (kvolkovich-sc) [reply src/client/components/DatePicker/DatePicker.react.js]: Fixed

File: src/client/components/DateRangeInput/DateRangeInput.react.js [hunk 5]
```diff
@@ -198,23 +200,25 @@ class DateRangeInput extends Component {
...
-        enteredTo: null
-      });
-    } else {
-      this.handleRangeChange([from, day]);
-      this.setState({
-        enteredTo: day,
-        showPicker: false
-      });
+  handleDayClick = (dayValue, modifiers) => {
+    if (!modifiers.disabled) {
```
Comment (estambakio-sc) [inline src/client/components/DateRangeInput/DateRangeInput.react.js]: Better 
```
if (modifiers.disabled) {
  return
}

... code
```

File: src/client/components/DayPicker/DayPicker.DOCUMENTATION.md (hunk 3 above)
Comment (kvolkovich-sc) [reply src/client/components/DayPicker/DayPicker.DOCUMENTATION.md]: No.

Changed to `modifiers: { disabled: ... }` Fixed.

**Pull request merged**